moviebot/controller/controller_telegram.py

```diff
@@ -33,11 +34,9 @@
 
 
 class ControllerTelegram(Controller):
-    """This is the Controller class which controls the flow of the conversation
-    while the user interacts with the agent using telegram."""
-
-    def __init__(self):
-        """Initializes some basic structs for the Controller."""
+    def __init__(self) -> None:
+        """Controller class which controls the flow of the conversation while
+        the user interacts with the agent using telegram."""
```

Comment:
Can we try to fit it in one line? For example, _Controller class to control the flow of the conversation on Telegram._

Reply:
done